### ISSUE TITLE
Cannot add/create Custom Components

### ISSUE THREAD

**Issue by McKBo** [FlutterFlow/flutterflow-issues#2451]
### Has your issue been reported?

- [X] I have searched the existing issues and confirm it has not been reported.
- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

When I try to **create a component** from anything in the Widget Tree, I see the pop-up to enter a name but cannot type there - though it seems like the input is received by the now faded out system in the background (I can navigate in the opened menu in the background using arrow keys).

When I try to **add an existing Custom Component** to a page in the Widget Tree, they flicker into the tree and out again. I cannot add them by dragging them into the screen I am working on, either.

I tried this in an existing larger project with previously functional Components, and in a new blank project.

### Expected Behavior

I would expect

* the Custom Component to be added to the page.
* being able to enter a name and a Custom Component to be created when using the right-click menu to create it

### Steps to Reproduce

To see fail in creation:
1 - Create a new project, create a new page
2 - add a few items to the Widget Tree
3 - try to turn them into a Custom Component.

*Or to see fail in adding component in a project that already has one:*
1 - open existing project with existing Custom Components
2 - try to add one to any page.

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT4siMn1z8pgoelJ1s7ubPlB/SA4JVh5U7gkscpCbw4bIYj2P4lzY876PhBtQumlTHxpeU2mp14xw/DUuOLLNfIBHyudfL57/pAJTAvhdzuiRqWTF7W8OkBSRdJhGEyG0Z6rsCN4PsFYLnwb706Qe+ytTgTXHqKwCmowSq/LZO4=

### Context

I cannot continue working on my projects using Custom Components.

### Visual documentation

![image](https://uploads.linear.app/3241089d-736a-4422-900b-051e72accdbe/c578f477-7228-4b11-867b-dd23d61a40b3/cc4f9e1d-e96f-4b60-822e-056dacda1508?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiLzMyNDEwODlkLTczNmEtNDQyMi05MDBiLTA1MWU3MmFjY2RiZS9jNTc4ZjQ3Ny03MjI4LTRiMTEtODY3Yi1kZDIzZDYxYTQwYjMvY2M0ZjllMWQtZTk2Zi00YjYwLTgyMmUtMDU2ZGFjZGExNTA4IiwiaWF0IjoxNzA5NzMxMjY5LCJleHAiOjMzMjgwMjkxMjY5fQ.lC2FaM8_ZCSQr-7N50yt9JYJ2UxHg7tGGNYZxm9SGq0)

As the only thing I can "show" here: you can see the changed chosen item in the menu in the background.

### Additional Info

The above Bug Report Code is for simplification from an empty, simple project. There, you can of course only reproduce "not being able to create component".

I tested the behaviour with different browsers and Windows versions; below one example.

I first noticed the problem Monday, March 4th and before that had last worked on the project without noticing problems on Friday, March 2nd.

### Environment

```markdown
- FlutterFlow version: v4.1, released Feb 24, 2024
- Platform: Web
- Browser name and version: Chrome 122.0.6261.95
- Operating system and version affected: Windows 10 Home 22H2 19045.4046
```

**Comment by ChristopheHvd** [comment]
Hi,
I think this issue is the same as the one reported [here](https://github.com/FlutterFlow/flutterflow-issues/issues/2387) as well.

**Comment by takashiff** [comment]
We can leave this issue open as well, since it raises another problem that isn't addressed in the other issue.

**Comment by takashiff** [comment]
@McKBo I'm currently unable to reproduce the 2nd issue where custom components are unusable. Could you outline the steps or share a video where this is occurring? Thanks

**Comment by McKBo** [comment]
@takashiff Thank you for looking into it.

@wenkaifan0720 
In the mean time, I "fixed" my problem with not being able to add the components by setting my project back to last Friday, when it worked. I guess that this means I actually changed something that broke it before. If I do that unknown thing again and can reproduce the problem, I will deliver a video. 

Until then:
The problem appeared any time I tried to add any of my Custom Components anywhere in a page. That includes removing a component and trying to add the same one again. The confusing part is that it happened across other projects open at the same time, which are also now fixed again.


I also found a "trick" for the other problem, which is basically clicking into the naming area again and again and at the same time pasting a prepared name from my clipboard. That somehow gets around the forced focus change.
